etna/packages/etna-js/spec/helpers.js

```diff
@@ -12,11 +12,24 @@ export const stubUrl = ({
   request,
   status = 200,
   headers = {},
-  host = 'http://localhost'
+  host = 'http://localhost',
+  url,
 }) => {
   let nocked;
 
-  console.log('Stubbing for', {request, path, verb, response}, '\nactive')
+  if (url) {
```

Comment:
Adding the ability to pass in url instead of host and path separately.  Some utility functions (timur's path functions) return full urls, so this is more convenient as a parameter.